src/lighthouse.js

```diff
@@ -1,3 +1,6 @@
+// prevent logger from prefixing a date when running in tty
+process.env.DEBUG_COLORS = 'true';
```

Comment:
Note that `process.env.VAR = '...'` sets the environment variable in the next plugins and/or `build.command` as well.
I am guessing this might not be intended. This could be fixed by restoring the previous value of this environment variable right after it's been used.
@erezrokah

Reply:
Good point, I'll change it